webapp/src/pages/FoodCheckoutConfirmation.jsx

```diff
@@ -49,53 +50,77 @@ export default function FoodCheckoutConfirmation() {
         <p className="mb-0">{t("food:confirmation_subtitle")}</p>
       </div>
       <LayoutContainer gutters top>
-        <h4 className="mb-3">{t("food:checkout_items_title")}</h4>
+        <h4 className="mb-3">{t("food:confirmation_my_order")}</h4>
         {items.map((p, idx) => (
           <Item key={idx} item={p} />
         ))}
+        {user.unclaimedOrdersCount !== 0 && (
+          <FormButtons
```

Comment:
Make this a regular button, using flexbox to lay it out as you normally would. This is not a form.

If you start to hijack something semantically distinct (like form buttons) for something different (like a link button as you have here), it makes refactoring and redesigning a much bigger headache.